server/jetstream_api.go

```diff
@@ -27,118 +28,121 @@ import (
 
 // Request API subjects for JetStream.
 const (
-	// JetStreamEnabled allows a user to dynamically check if JetStream is enabled for an account.
-	// Will return +OK on success, otherwise will timeout.
-	JetStreamEnabled = "$JS.ENABLED"
+	// JSApiInfo is for obtaining general information about JetStream for this account.
+	// Will return JSON response.
+	JSApiAccountInfo = "$JS.API.INFO"
 
-	// JetStreamInfo is for obtaining general information about JetStream for this account.
+	// JSApiCreateTemplate is the endpoint to create new stream templates.
 	// Will return JSON response.
-	JetStreamInfo = "$JS.INFO"
+	JSApiTemplateCreate  = "$JS.API.TEMPLATE.CREATE.*"
+	JSApiTemplateCreateT = "$JS.API.TEMPLATE.CREATE.%s"
 
-	// JetStreamCreateTemplate is the endpoint to create new stream templates.
-	// Will return +OK on success and -ERR on failure.
-	JetStreamCreateTemplate  = "$JS.TEMPLATE.*.CREATE"
-	JetStreamCreateTemplateT = "$JS.TEMPLATE.%s.CREATE"
+	// JSApiListTemplates is the endpoint to list all stream templates for this account.
+	// Will return JSON response.
+	JSApiTemplates = "$JS.API.TEMPLATES"
```

Comment:
This feels wrong to me

```
$JS.API.TEMPLATE.CREATE.*
$JS.API.TEMPLATES
$JS.API.TEMPLATE.INFO.*
$JS.API.TEMPLATE.DELETE.*
```

The `TEMPLATES` should be `TEMPLATE.LIST` its more consistent and easier to block all access to template related things in a single ACL - ditto for the others

Reply:
That might be fair. The `TEMPLATE` ones all have an identifier, and when I had `TEMPLATE.LIST` the way I was looking at it that one stuck out.

I can go either way though.

Reply:
Will change all back to LIST form.